core-web/libs/portlets/edit-ema/portlet/src/lib/dot-ema-shell/dot-ema-shell.component.ts

```diff
@@ -78,6 +79,9 @@ import { NavigationBarItem } from '../shared/models';
 export class DotEmaShellComponent implements OnInit, OnDestroy {
     @ViewChild('dialog') dialog!: DotEmaDialogComponent;
     @ViewChild('pageTools') pageTools!: DotPageToolsSeoComponent;
+
+    private readonly didTranslate = signal(false);
```

Comment:
@zJaaal According to recents conversations in Slack, all signals should start with `$`, like this `$didTranslate`.